sample-app/src/main/AndroidManifest.xml

```diff
@@ -6,6 +6,10 @@
     <uses-permission android:name="android.permission.INTERNET"/>
     <uses-permission android:name="android.permission.ACCESS_BACKGROUND_LOCATION" />
 
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION"/>
+    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION"/>
```

Comment:
Did we always need these permissions? Just wondering if this is a new requirement, or if these were always required but for some reason we never added them.